components/base_page.js

```diff
@@ -7,14 +8,21 @@ import CommonJS from '../components/js/common';
 import NonBlockingJS from '../components/js/nonblocking';
 import Contact from '../components/contact';
 
+import PAGE_TITLES from '../constants/page_titles';
+import CONSTANTS from '../components/constants';
```

Comment:
These are both referring to constants.

We should just consolidate into one file, one single `constants.js` at the top-level.

(Also, note that this file is inside the `components/` directory already, so `../components/constants` is just `./constants`)